docs/internals/index.rst

```diff
@@ -8,3 +8,4 @@ Project Internals
    contributing
    releases
    roadmap
+   signatures
```

Comment:
I can easily imagine you'd prefer having this somewhere else. Does anybody have a good suggestion?